rabit/rabit/Goal/Models/Days.swift

```diff
@@ -42,32 +42,32 @@ struct Days: CustomStringConvertible {
         } else if isWeekdaysOnly {
             return "평일"
         } else {
-            return  Array(set).sorted { $0.rawValue < $1.rawValue}.description
+            return  Array(selectedValues).sorted { $0.rawValue < $1.rawValue}.description
```

Comment:
```suggestion
            return selectedValues.sorted { $0.rawValue < $1.rawValue }.description
```

이 부분 궁금해서 한 번 찾아봤는데, Set 타입 속성도 `sorted(by:)` 메소드를 거치면 Array로 자동 변환이 되는 것 같습니다!

그리고 Day 타입이 **Comparable** 프로토콜을 채택하면 해당 sorted 부분도 `sorted(by: <)` 로 조금 더 깔끔하게 고칠 수 있지 않을까 생각들었습니다ㅎㅎ

Reply:
와 여기서 바로 제안한거 커밋 가능하네요 ㅋㅋ 
제안해주신 부분이 훨씬 더 깔끔한 것 같습니다! 개선해서 커밋 푸시할께요 ㅎㅎ